templates/OperationRequestPatchBodySnippet.stencil

```diff
@@ -0,0 +1,21 @@
+// Construct patch request
+let patch = MergePatchObject()
+{% for name in op.request.bodyParam.propertyNames %}
+if let val = {{ op.request.bodyParam.name }}.{{ name }} {
+    patch.replace(atPath: "{{name}}/", withValue: val)
```

Comment:
so we only support replace now? others like 'add', 'remove' is not going to be supported?

Reply:
Where in swagger/code model can tell us whether it is a 'patch.replace' instead of 'patch.add' or 'patch.remove'?

Reply:
All of the methods are in the PatchUtil file, but I don't see any of the actual SDK patch scenarios using anything other than `replace`. To use any of the others, we'd need to know what the original object looked like and diff them, which I would really not want to do.